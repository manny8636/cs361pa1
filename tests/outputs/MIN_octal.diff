--- outputs/MIN_octal.txt	2022-02-14 17:13:34.972907867 -0500
+++ expected/MIN_octal.txt	2021-01-25 15:27:24.000000000 -0500
@@ -1 +1,2 @@
-Parsing ../data/octal.txt failed
+VALUE [integer]: 16
+Success!
