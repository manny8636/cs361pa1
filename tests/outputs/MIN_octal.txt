Parsing ../data/octal.txt failed
