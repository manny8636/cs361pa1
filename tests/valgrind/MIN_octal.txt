==3062710== Memcheck, a memory error detector
==3062710== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==3062710== Using Valgrind-3.15.0 and LibVEX; rerun with -h for copyright info
==3062710== Command: ../json -v ../data/octal.txt
==3062710== 
Success!
==3062710== 
==3062710== HEAP SUMMARY:
==3062710==     in use at exit: 4 bytes in 1 blocks
==3062710==   total heap usage: 4 allocs, 3 frees, 16,860 bytes allocated
==3062710== 
==3062710== LEAK SUMMARY:
==3062710==    definitely lost: 4 bytes in 1 blocks
==3062710==    indirectly lost: 0 bytes in 0 blocks
==3062710==      possibly lost: 0 bytes in 0 blocks
==3062710==    still reachable: 0 bytes in 0 blocks
==3062710==         suppressed: 0 bytes in 0 blocks
==3062710== Rerun with --leak-check=full to see details of leaked memory
==3062710== 
==3062710== For lists of detected and suppressed errors, rerun with: -s
==3062710== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
