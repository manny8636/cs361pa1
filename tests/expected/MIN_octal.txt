VALUE [integer]: 16
Success!
